composer.json

```diff
@@ -9,28 +9,31 @@
     ],
     "license": "MIT",
     "require": {
-        "php": "^7.3",
+        "php": "^7.2.5",
         "ext-json": "*",
         "ext-simplexml": "*",
+        "fideloper/proxy": "^4.2",
+        "fruitcake/laravel-cors": "^1.0",
         "bensampo/laravel-enum": "^1.26",
+        "guzzlehttp/guzzle": "^6.3",
         "bonroyage/oauth": "1.*",
-        "laravel/framework": "^6.8",
-        "laravel/passport": "^7.3",
-        "laravel/tinker": "^1.0",
+        "laravel/framework": "^7.0",
+        "laravel/passport": "^8.4",
+        "laravel/tinker": "^2.0",
         "mll-lab/graphql-php-scalars": "^2.1",
         "nuwave/lighthouse": "^4.7",
```

Comment:
This still good?

Reply:
Current highest is 4.11, this should be captured in the lock file anyway

Reply:
Assuming we don’t need a min of that then?

Reply:
Fair point, changed